Sources/CartonCLI/Commands/Test.swift

```diff
@@ -56,6 +56,9 @@ struct Test: AsyncParsableCommand {
   )
   var host = "127.0.0.1"
 
+  @Option(help: "Use the given bundle instead of building the test target")
+  var bundlePath: String?
```

Comment:
Would you consider something like `prebuiltBinaryPath` here? It's not clear what exactly "bundle" means in this context. It took me some time to remember that XCTest names these as test bundles, but I think "prebuilt binary" or something like that is much more clear.

Reply:
Yeah, actually the produced wasm binary is not "bundle" in darwin terms, and I agree it's unclear here...

`--prebuilt-binary-path` sounds better to me but not the best since "binary" is a little bit ambiguous.
But let's go ahead with `--prebuilt-binary-path` and rename it after we will come up with a good idea.

Reply:
`--prebuilt-test-bundle-path` could also work maybe? But still confusing in the context of `carton bundle`, so maybe `--prebuilt-binary-path` really is the best currently known option?

Reply:
Another way could be to pass it to `carton test directly`, without options: `carton test test.wasm`

Reply:
A little bit long, but sounds better 👍

Reply:
WDYT about passing to `carton test` directly, like `carton test test.wasm`?

Reply:
I thought it first, but positional arguments are already used as `testCases`